src/Components/ReflectionForm/ReflectionForm.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './ReflectionForm.scss';
+
+const ReflectionForm = () => {
+
+  return (
+    <section>
+      <h1>Reflection Form</h1>
+      <Link to='/myreflections'>
```

Comment:
Glad to see we got this path, written out, even when we don't have the rest of the form written out yet.